test/helpers/messages/test.js

```diff
@@ -22,8 +22,8 @@ define(['lodash', 'taoQtiTest/runner/helpers/messages'], function(_, messagesHel
     'use strict';
 
     var messagesHelperApi = [{ title: 'getExitMessage' }];
-    var testActionMessage = 'You will not be able to access this test once submitted. Click \"OK\" to continue and submit the test.';
-    var actionMessage = 'Click \"OK\" to continue.';
+    var testActionMessage = 'You will not be able to access this test once submitted. Click "OK" to continue and submit the test.';
```

Comment:
In this case - key will be changed, and translations will be lost. Also could be case that it is used in ManualTranslations and for some customer it will be lost

Reply:
sorry, didn't notice :)